ShellScript/Bash.sublime-syntax

```diff
@@ -1958,6 +1958,8 @@ contexts:
       scope: constant.character.escape.shell
 
   string-escapes:
+    - match: '\\(?:e|033)\[\d{1,3}(?:;\d{1,3}){0,2}m'
```

Comment:
The color sequence can go up to 24-bit `ESC[ 38;2;⟨r⟩;⟨g⟩;⟨b⟩ m`.
https://en.wikipedia.org/wiki/ANSI_escape_code#24-bit

Reply:
🤷‍♂️ If you prefer it, you can make a suggestion. If you really want, we can even [expressly make 0-255](https://regex101.com/r/BAjN4w/1).

Reply:
No. I mean your regex didn't highlight something like `\033[38;2;0;255;0m`

Reply:
```suggestion
    - match: '\\(?:e|033)\[\d{1,3}(?:;\d{1,3}){0,4}m'
```